### PR TITLE
Sommelier-4

### PR DESCRIPTION
**Pull request by Maximus135**
add fixes

### CODE REVIEW

File: App/components/SearchInput/input.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import {
+  NativeSyntheticEvent,
+  TextInputChangeEventData,
+  TouchableOpacity,
+} from 'react-native'
+import SvgUri from 'react-native-svg-uri'
```
Comment (Mixxx-a) [inline App/components/SearchInput/input.tsx]: delete SvgUri

File: App/components/SearchInput/input.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <Close
+              width={theme.inputSearchFontSizes[size]}
+              height={theme.inputSearchFontSizes[size]}
+            />
+          </TouchableOpacity>
+        </Styled.ViewClose>
+      )}
+    </Styled.StyledView>
+  )
+}
```
Comment (Mixxx-a) [inline App/components/SearchInput/input.tsx:63]: add export default SearchInput

File: App/components/SearchInput/input.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+type Props = {
+  defaultValue?: string
+  placeholder?: string
+  editable: boolean
+  size: keyof typeof theme.inputSearchSizes
+  maxLength?: number
+  value?: string
+  onChange: (e: NativeSyntheticEvent<TextInputChangeEventData>) => void
+  clearButtonMode: boolean
```
Comment (Mixxx-a) [inline App/components/SearchInput/input.tsx]: Shouldn't all props be optional?

**Review by Mixxx-a — APPROVED** [review]

File: App/components/SearchInput/input.tsx (hunk 2 above)
Comment (Maximus135) [reply App/components/SearchInput/input.tsx:63]: done

File: App/components/SearchInput/input.tsx (hunk 1 above)
Comment (Maximus135) [reply App/components/SearchInput/input.tsx]: done

File: App/components/SearchInput/input.tsx (hunk 3 above)
Comment (Maximus135) [reply App/components/SearchInput/input.tsx]: done

**Review by StanislavMyakishev — APPROVED** [review]

**Pull request merged**